### ISSUE TITLE
Debian GNU/Linux 11 shim-15.6-1 x64 and ia32

### ISSUE THREAD

**Issue by steve-mcintyre** [rhboot/shim-review#268]
Confirm the following are included in your repo, checking each box:

 - [x] completed README.md file with the necessary information
 - [x] shim.efi to be signed
 - [x] public portion of your certificate(s) embedded in shim (the file passed to VENDOR_CERT_FILE)
 - [x] binaries, for which hashes are added to vendor_db ( if you use vendor_db and have hashes allow-listed )
 - [x] any extra patches to shim via your own git tree or as files
 - [x] any extra patches to grub via your own git tree or as files
 - [x] build logs
 - [x] a Dockerfile to reproduce the build of the provided shim EFI binaries

-------------------------------------------------------------------------------
### What is the link to your tag in a repo cloned from rhboot/shim-review?
-------------------------------------------------------------------------------
`https://github.com/steve-mcintyre/shim-review/tree/debian-11-shim-amd64-i386-20220806`

-------------------------------------------------------------------------------
### What is the SHA256 hash of your final SHIM binary?
-------------------------------------------------------------------------------
```
15a08b2166efb2141e0d21e77b2fb61b9005dc6f80c8ae0ae91f1bbb9048dc0b  shimia32.efi
0f8db91f4e202302e87cb6dd225b46258733758e0c9a44deda6f91b68df718b3  shimx64.efi
```

**Comment by ecos-platypus** [comment]
Disclaimer: I am not an authorized reviewer but review other shims to reduce the workload of the authorized reviewers and speed up the process for everyone.

Review was conducted in accordance to the reviewer guidelines (https://github.com/rhboot/shim/wiki/reviewer-guidelines)

This request is very similar to #267 which I reviewed before (https://github.com/rhboot/shim-review/issues/267#issuecomment-1207699923). I diffed the repositories associated with both requests and only highlight the differences.

- Build reproduces for `shimx64.efi` and `shimia32.efi`
- shim is built from https://salsa.debian.org/efi-team/shim.git rather than the release tar.gz. I diffed the `15.6` tag in the upstream shim repository with the `debian/15.6-1_deb11u1` tag checked out from the Debian repository and there were no changes in the shim source code apart from an additional `debian` directory
  - Two patches are applied for `arm64` for builds with older toolchains but as outlined in README.md, the `arm64` binaries do not need to be signed as part of this shim review request
- Linux Kernel 5.10.127 with lockdown patches is used

Looks good to me.
